src/test/public-server.ts

```diff
@@ -18,6 +19,9 @@ export const getServer = () => {
     schema: buildSubgraphSchema([{ typeDefs: typeDefsPublic, resolvers }]),
     context: {
       db: client(),
+      dataLoaders: {
+        collectionLoader,
+      },
```

Comment:
how was this working previously without `dataLoaders` defined here?

Reply:
The other tests in this file were testing functions that never used a dataloader, just direct db queries!